static/js/copy_and_paste.js

```diff
@@ -35,6 +35,19 @@ function find_boundary_tr(initial_tr, iterate_row) {
     return [rows.id(tr), skip_same_td_check];
 }
 
+function check_blocks(start_id, end_id) {
```

Comment:
We can name this check_multiple_messages_selected.

But I'm surprised we don't already have such a function, since we need to check for this anyway for the current behavior?